### PR TITLE
8 ticket patch api articles article

### PR DESCRIPTION
**Pull request by Anna702**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -277,3 +277,114 @@ describe("POST: /api/articles/:article_id/comments", () => {
...
+      .then(({ body }) => {
+        expect(body.msg).toBe(
+          "Bad request: inc_votes is required and must be a number"
+        );
+      });
+  });
+
+  test("400: PATCH responds with right message when article.votes would become a negative number", () => {
+    const incVotes = {
+      inc_votes: -1000,
```
Comment (pippaamy) [inline __tests__/app.test.js:408]: I think this is a sad path test!

File: controllers/articles_controller.js [hunk 2]
```diff
@@ -20,3 +24,15 @@ exports.getArticleById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.patchArticleVotes = (req, res, next) => {
+  const { article_id } = req.params;
+  const inc_votes = req.body.inc_votes;
+  updatedArticleVotes(article_id, inc_votes)
+    .then((article) => {
+      res.status(200).send(article);
```
Comment (pippaamy) [inline controllers/articles_controller.js]: send this back on a key of article instead -> {article}

**Review by pippaamy — COMMENTED** [review]
Good work Anna, just a couple of changes to make!

File: controllers/articles_controller.js (hunk 2 above)
Comment (Anna702) [reply controllers/articles_controller.js]: can you clarify this please? we need to send back the updated article - it is exactly what we have now...

**Review by Keb84 — APPROVED** [review]
Ready to merge 🎉

**Pull request merged**